package.json

```diff
@@ -4,6 +4,9 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "build-babel": "npm run clean && babel ./ -d dist/ --ignore node_modules",
+    "clean": "rimraf dist/",
```

Comment:
Please why are we using `rimraf` @macphilips

Reply:
I'm using `rimraf` to remove the dist directory before using babel to transpiler